cnchi/installation/install.py

```diff
@@ -774,10 +774,10 @@ def update_pacman_conf():
                     else:
```

Comment:
Let's do this instead: 
```python
elif pacline == '[core]\n':
    pacman_file.write('\n\n#[antergos-staging]\n')
    pacman_file.write('#SigLevel = PackageRequired\n')
    pacman_file.write('#Include = /etc/pacman.d/antergos-mirrorlist\n\n')
    pacman_file.write('[antergos]\n')
    pacman_file.write('SigLevel = PackageRequired\n')
    pacman_file.write('Include = /etc/pacman.d/antergos-mirrorlist\n\n')
    pacman_file.write(pacline)
else:
    pacman_file.write(pacline)
```
Its kind of ugly but since I am trying to write this on github as feedback for you instead of doing it myself in the code its the best I can do :wink: